canonical-web-and-design/maas.io
# Add discourse module to project + set up basic templates

## Done
- Added canonicalwebteam.discourse_docs to project.
- Set up basic docs styles and templates.

## QA
- Go to http://0.0.0.0:8006/docs
- Check that content is being pulled from the [MAAS docs discourse](https://discourse.maas.io/c/docs)
- **Note:** there are quite a few issues with the MAAS discourse forum, which I've listed [here](https://docs.google.com/spreadsheets/d/1JdhzqdXy_DqCDJR40Gj09ovSNWIWldiz8VoIEPVP9zE/edit#gid=0)

Fixes canonical-web-and-design/MAAS-squad#1299
Fixes canonical-web-and-design/MAAS-squad#1300
